tools/vault/base/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - statefullset.yaml
+  - service.yaml
+  - ingress.yaml
+
+namespace: fake-namespace
```

Comment:
I assume this is 'fake' as in placeholder value which is overwritten later in the (overlay) proces? Maybe placeholder is a better name then.

Reply:
processed